.docker/aiida-core-with-services/s6-assets/init/rabbitmq-init.sh

```diff
@@ -14,8 +14,8 @@ echo MNESIA_BASE="${RABBITMQ_DATA_DIR}" >> "${RMQ_ETC_DIR}/rabbitmq-env.conf"
 echo LOG_BASE="${RABBITMQ_DATA_DIR}/log" >> "${RMQ_ETC_DIR}/rabbitmq-env.conf"
 
 # using workaround from https://github.com/aiidateam/aiida-core/wiki/RabbitMQ-version-to-use
-# set timeout to 100 hours
-echo "consumer_timeout=3600000" >> "${RMQ_ETC_DIR}/rabbitmq.conf"
+# 1000 hours in milliseconds
+echo "consumer_timeout=3600000000" >> "${RMQ_ETC_DIR}/rabbitmq.conf"
```

Comment:
Wouldn't it perhaps be better to add the `advanced.confgi` approach instead, which allows us to simply set the timeout to undefined? I think the following should work
```suggestion
cat > "${RMQ_ETC_DIR}/advanced.config" <<EOF
%% advanced.config
[
  {rabbit, [
    {consumer_timeout, undefined}
  ]}
].
EOF
```

Reply:
Ah, cool, I didn't know this. Will give it a build locally and test.

Reply:
Changed and add test checking it is set, but better to run a calculation to test it really activated.